PdfReader/Form1.cs

```diff
@@ -164,28 +167,61 @@ private void FillPages(RichTextBox richTextBox, string text)
                 }
             }
 
-           
+
             pages.Add($"{Environment.NewLine}{currentPageText.ToString()}{Environment.NewLine}");
-            if(pages.Count<=page)
+            if (pages.Count <= page)
             {
                 page -= 1;
             }
+
             richTextBox1.Text = pages[page];
             label1.Text = page.ToString();
             label2.Text = "/";
-            label2.Text += currentPage-1;
+            label2.Text += currentPage - 1;
+            UpdateButtonSizeAndPosition();
         }
 
         private void Form1_SizeChanged(object sender, EventArgs e)
         {
-            if(pageText.Length > 0) {
+            UpdateButtonSizeAndPosition();
+            if (pageText.Length > 0)
+            {
                 FillPages(richTextBox1, pageText);
             }
             else
             {
                 return;
             }
-           
+        }
+
+        private void UpdateButtonSizeAndPosition()
```

Comment:
it is more preferred to use anchors in form constractor, but ok, delete coment from chatgpt and reopen pull request